app/mailers/mailer.rb

```diff
@@ -1,11 +1,8 @@
-class CommunicartMailer < ApplicationMailer
+class Mailer < ApplicationMailer
```

Comment:
`ApplicationMailer` would be a good choice for the base mailer. Matches with `ApplicationController`, `ApplicationJob`, and `ApplicationHelper`. :blush: